exercises/input-output/src/main/java/com/cbfacademy/WriteStream.java

```diff
@@ -0,0 +1,35 @@
+package com.cbfacademy;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class WriteStream {
+    public void writeStream() throws IOException{
+        String filePath = "src/main/resources/exercise.txt";
+        // input filepath
+        String outputPath = "src/main/resources/writingExercise.txt";  
+        // output filepath
+        try { 
+            File inputFile = new File(filePath);
+            File outputFile = new File(outputPath);
+
+            FileInputStream readingFile = new FileInputStream(inputFile);
+            FileOutputStream writingFile = new FileOutputStream(outputFile); 
+
+            int i;
+
+            while((i = readingFile.read()) != -1){
+                writingFile.write(i);
+            }
+            readingFile.close();
+            writingFile.close();
+            
+        } catch (Exception e) {
```

Comment:
do we want to do anything here once the exception is caught?

Reply:
Also, if you refer to our exception handling session, what's the danger of using the `Exception` class here?